occurrences/models.py

```diff
@@ -435,7 +435,10 @@ def delete(self, *args, **kwargs):
         self.__post_delete_unpublish_event()
 
     def __str__(self):
-        return f"{self.p_event.linked_event_id} {self.start_time} {self.place_id}"
+        if self.p_event:
+            return f"Event {self.p_event.linked_event_id} at {self.place_id} ({self.start_time.strftime('%Y-%m-%d %H:%M')})"  # noqa: E501
+        else:
+            return f"Event at {self.place_id} ({self.start_time.strftime('%Y-%m-%d %H:%M')})"  # noqa: E501
```

Comment:
One alternative to get rid of the long lines:
```suggestion
        event = f"Event {self.p_event.linked_event_id}" if self.p_event else "Event"
        return (
            f"{event} at {self.place_id} ({self.start_time.strftime('%Y-%m-%d %H:%M')})"
        )
```

Reply:
If using the noqa: E501 is a problem here, I would be more willing to replace this with string format-command instead.

Reply:
Just thought I'd get rid of the `# noqa: E501`'s and long lines. If you have other ideas how to make do without them, go ahead.